src/libraries/Microsoft.Extensions.Options/src/OptionsServiceCollectionExtensions.cs

```diff
@@ -134,7 +134,8 @@ public static IServiceCollection PostConfigure<TOptions>(this IServiceCollection
             => services.PostConfigure(name: null, configureOptions: configureOptions);
 
         /// <summary>
-        /// Registers a type that will have all of its I[Post]ConfigureOptions registered.
+        /// Registers a type that will have all of its IConfigureOptions, IPostConfigureOptions,
+        /// and IValidateOptions registered.
```

Comment:
What do y'all think of tweaking this documentation to something like this?

```suggestion
        /// Registers the implementations of <see cref="IConfigureOptions{TOptions}"/>,
        /// <see cref="IPostConfigureOptions{TOptions}"/>, and <see cref="IValidateOptions{TOptions}"/>
        /// from type <typeparamref name="TConfigureOptions"/>.
```